src/cli/argument-parser/index.ts

```diff
@@ -202,6 +202,7 @@ export default class CLIArgumentParser {
             .option('--disable-http2', 'force the proxy to issue HTTP/1.1 requests')
             .option('--cache', 'cache web assets between test runs')
             .option('--base-url <url>', 'set the base url for the test run')
+            .option('--login', 'login in TestCafe')
```

Comment:
Can we add the `login` command (not an option)? Like we did for dashboard https://github.com/DevExpress/testcafe/pull/6975